platform/darwin/src/MGLOfflinePack.mm

```diff
@@ -207,6 +214,8 @@ - (void)offlineRegionStatusDidChange:(mbgl::OfflineRegionStatus)status {
 }
 
 - (void)didReceiveError:(NSError *)error {
+    MGLLogInfo(@"Notifying pack error.");
+    MGLLogError(@"Error: %@", error.localizedDescription);
```

Comment:
Why are there two logs here? Can we combine?

Reply:
One specifies the action that is going to take place, and the other the error that is receiving.

Reply:
Perhaps we might flip these two lines, then, as that would be more reflective of the order in which things are happening (i.e., there was an error, then we sent a notification about that error).

Reply:
The grammar of “Notifying pack error” is also a bit ambiguous — it could be read to mean that you’re sending a notification _to_ the pack error. Adding an “about” might make this more clear.